azure-pipeline-codeql.yml

```diff
@@ -0,0 +1,155 @@
+        demands: ImageOverride -equals Build.Server.Amd64.VS2019
+      displayName: "CodeQL Scan"
+      steps:
+
+      # Guardian requirements
+      - task: NuGetToolInstaller@1
+        displayName: 'Install NuGet.exe'
+
+      - task: NuGetAuthenticate@0
+        displayName: 'Authenticate NuGet'
+        inputs:
+          nuGetServiceConnections: GuardianConnect
+  
+      - pwsh: |
+          . $(Build.SourcesDirectory)\eng\CodeQL.ps1
+          $guardianCliLocation = Install-Gdn -Path $(Build.SourcesDirectory)\.artifacts
+          Write-Host "##vso[task.setvariable variable=GuardianCliLocation]$guardianCliLocation"
+        displayName: Install Guardian
+
+      # Project requirements
+      - task: UseDotNet@2
+        displayName: Install Correct .NET Version
+        inputs:
+          useGlobalJson: true
+
+      - task: UseDotNet@2
+        displayName: Install .NET Version 3.1
+        inputs:
+          version: 3.1.x
+
+      - task: NodeTool@0
+        displayName: 'Install Node'
+        inputs:
+          versionSpec: '12.x'
+
+      - task: NuGetCommand@2
+        displayName: Restore Packages
+        inputs:
+          command: restore
+          solution: "**/*.sln"
+          feedstoUse: config
+
+      - pwsh: eng\set-version-parameters.ps1
+        displayName: Calculate release version variables
+        
+      - pwsh: |
+          [xml]$manifest = Get-Content src\Maestro\MaestroApplication\ApplicationPackageRoot\ApplicationManifest.xml
+          $manifest.SelectSingleNode("/*[local-name()='ApplicationManifest']/*[local-name()='Policies']").RemoveAll()
+          $manifest.SelectSingleNode("/*[local-name()='ApplicationManifest']/*[local-name()='Principals']").RemoveAll()
+          $manifest.Save("src\Maestro\MaestroApplication\ApplicationPackageRoot\ApplicationManifest.xml")
+          git diff
+        displayName: Remove Service Fabric RunAsPolicy from MaestroApplication
+
+      - pwsh: |
+          [xml]$manifest = Get-Content src\Telemetry\TelemetryApplication\ApplicationPackageRoot\ApplicationManifest.xml
+          $manifest.SelectSingleNode("/*[local-name()='ApplicationManifest']/*[local-name()='Policies']").RemoveAll()
+          $manifest.SelectSingleNode("/*[local-name()='ApplicationManifest']/*[local-name()='Principals']").RemoveAll()
+          $manifest.Save("src\Telemetry\TelemetryApplication\ApplicationPackageRoot\ApplicationManifest.xml")
+          git diff
+        displayName: Remove Service Fabric RunAsPolicy from TelemetryApplication
+      
+      - pwsh: |
+          . $(Build.SourcesDirectory)\eng\codeql.ps1
+          Initialize-Gdn -GuardianCliLocation $(GuardianCliLocation) `
+            -WorkingDirectory $(Build.SourcesDirectory) `
+            -LoggerLevel 'Standard'
+        displayName: 'CodeQL: Initialize'
+
+      - pwsh: |
+          . $(Build.SourcesDirectory)\eng\codeql.ps1
+          New-GdnSemmelConfig -GuardianCliLocation $(GuardianCliLocation) `
```

Comment:
Some places, it's Semmle, here it's Semmel... looks like a function naming problem

Reply:
Yes, my spelling is bad apparently. Will fix.